aiida/plugins/entry_point.py

```diff
@@ -30,11 +30,18 @@
 ENTRY_POINT_STRING_SEPARATOR = ':'
 
 
-@functools.lru_cache(maxsize=1)
-def eps():
+@functools.cache
+def eps() -> EntryPoints:
     return _eps()
 
 
+@functools.lru_cache(maxsize=100)
+def eps_select(group, name=None) -> EntryPoints:
```

Comment:
Can you please add a docstring here and add a note that you are providing this over `eps.select` in order to be able to cache the calls